kloppy/infra/serializers/tracking/epts/meta_data.py

```diff
@@ -57,18 +67,22 @@ def _load_periods(global_config_elm, frame_rate: int) -> List[Period]:
     return periods
 
 
-def _load_players(players_elm, team_map: Dict[str, Team]) -> List[Player]:
+def _load_players(players_elm, team: Team) -> List[Player]:
     return [
         Player(
-            team=team_map[player_elm.attrib["teamId"]],
+            team=team,
```

Comment:
Now we are adding to each Player, the full Team object (which already contains all the players). Not sure if this is correct, or if we should add only the team_id instead of the full team object.

Reply:
This will be a reference to the Team object instead of a full copy so that's fine.